AlphaWalletTests/Tokens/Coordinators/GetIsERC1155ContractCoordinatorTestCase.swift

```diff
@@ -30,13 +30,18 @@ class GetIsERC1155ContractCoordinatorTestCase: XCTestCase {
     func testgetIsERC1155Contract() throws {
         let coordinator = GetIsERC1155ContractCoordinator(forServer: .main, cacheName: fileName)
         let expectation = expectation(description: "Waiting for server response")
-        coordinator.getIsERC1155Contract(for: address1)
-            .done { returnValue in
-                self.result = returnValue
-                expectation.fulfill()
-            }
-            .cauterize()
-        wait(for: [expectation], timeout: 10 * 60)
+        var fetchedError: Error?
+        firstly {
+            coordinator.getIsERC1155Contract(for: address1)
+        }.done { returnValue in
+            self.result = returnValue
+        }.ensure {
+            expectation.fulfill()
+        }.catch {error in
+            fetchedError = error
+        }
+        wait(for: [expectation], timeout: 20)
+        XCTAssertNil(fetchedError, "Unknown error: \(fetchedError!)")
```

Comment:
How about moving the assert up into the `catch {}`? Merge when ready.

Reply:
I prefer to do all the asserts after an expectation is fulfilled.